src/components/Questions/QuestionModal.vue

```diff
@@ -168,7 +169,7 @@ export default defineComponent({
 
         // if the selection option was already in the response
         // remove it from the response (uncheck it); otherwise add it (check it)
-        const currentResponse = JSON.parse(JSON.stringify(state.draftResponses[props.currentQuestionIndex]))
+        const currentResponse = clonedeep(state.draftResponses[props.currentQuestionIndex])
         // JSON parse + stringify clones the array (which may contain any complex object; responses here)
```

Comment:
this comment was not updated. Also, ideally, this comment should come on top of the line above, not below